src/metabase/models/card.clj

```diff
@@ -68,7 +68,7 @@
 (defmethod mi/can-write? Card
   ([instance]
    ;; Cards in audit collection should be read only
-   (if (perms/is-parent-collection-audit? instance)
+   (if (= (:collection_id instance) (:id (perms/default-audit-collection)))
```

Comment:
on oss this is failing for every item in the root collection (`collection_id` is nil).

```clojure
dashboard=> (perms/default-audit-collection)
:metabase.models.permissions/noop
dashboard=> (:id (perms/default-audit-collection))
nil
```

We need to check that `(= perms/audit-db-id (:collection_id instance))` or we could check if the audit collection exists. But nil is a valid value that indicates root collection.

Reply:
or of course just `(and (:id (perms/default-audit-collection)) (= (:id (perms/default-audit-collection)) ...))`

Reply:
Oh, so this was making every card in the root collection to be read-only on OSS. Let's just use the id itself instead.